README.md

```diff
@@ -249,25 +251,24 @@ Help improve Public Lab software!
 * Join the 'plots-dev@googlegroups.com' discussion list to get involved
 * Look for open issues at https://github.com/publiclab/mapknitter/issues
 * Review contributor guidelines at http://publiclab.org/wiki/contributing-to-public-lab-software
-* Some devs hang out in http://publiclab.org/chat (irc webchat)
+* Some devs hang out in http://publiclab.org/chat (IRC webchat)
 * Find lots of info on contributing at http://publiclab.org/wiki/developers
 * Join our gitter chat at https://gitter.im/publiclab/publiclab
 
 ## Staging infrastructure and testing
 
-In addition automatic testing with Travis CI - we have a branch (`unstable`) is set to auto-build and deploy to [a staging instance](http://mapknitter-unstable.laboratoriopublico.org/). This instance includes a copy of the production database and is intended for experimenting or debugging purposes with a production-like environment. We also have a `stable` build at http://mapknitter-stable.laboratoriopublico.org/ which builds off of our `main` branch. Any commits or PRs merged to the main branch will trigger the `stable` server to rebuild; you can monitor progress at https://jenkins.laboratoriopublico.org/
+In addition to automatic testing with Travis CI, we have a branch (`unstable`) that is set to auto-build and deploy to [a staging instance](http://mapknitter-unstable.laboratoriopublico.org/). This instance includes a copy of the production database and is intended for experimenting or debugging purposes in a production-like environment. We also have a `stable` build at http://mapknitter-stable.laboratoriopublico.org/ which builds off of our `main` branch. Any commits or PRs merged to the main branch will trigger the `stable` server to rebuild, and you can monitor the progress at https://jenkins.laboratoriopublico.org/
 
 ****
 
 ## License
 
-MapKnitter is free software: you can redistribute it and/or modify
+MapKnitter is a free software: you can redistribute it and/or modify
```

Comment:
Wow this is super nice! I wanted to note that this text can't be edited - it's part of legal language related to the GPL, so has to be included exactly.

Once we revert this, the rest is good to merge, thank you!

Reply:
Oh! Alright. Would revert this section then. Thank you 😄